mysql-test/main/auto_increment.test

```diff
@@ -433,3 +439,24 @@ SET @engine='MyISAM';
 --echo # End of 10.2 tests
 --echo #
 
+--echo #
+--echo # MDEV-32018 Allow the setting of Auto_increment on FK referenced columns
+--echo #
+
+CREATE TABLE t1 (
+  id int unsigned NOT NULL PRIMARY KEY
+);
+
+CREATE TABLE t2 (
```

Comment:
This file doesn't use innodb at all. I think that better to move the test somewhere into `innodb` suite. A good pick can be `foreign_key.test`. I check other options like alter and autoinc there in the suite, but they are mostly debug-only, unlike `foreign_key`.